File: contact_me.php
<?php

require_once 'google/appengine/api/mail/Message.php';

use google\appengine\api\mail\Message;

// check if fields passed are empty
if(empty($_POST['name'])                  ||
   empty($_POST['email'])                 ||
   empty($_POST['message'])        ||
   !filter_var($_POST['email'],FILTER_VALIDATE_EMAIL))
   {
        echo "No arguments Provided!";
        return false;
   }
        
$name = $_POST['name'];
$email_address = $_POST['email'];
$message = $_POST['message'];
        
// create email body and send it        
$to = 'me@ulysseszheng0.appspotmail.com'; // put your email
$email_subject = "Contact form submitted by:  $name";
$email_body = "You have received a new message. \n\n".
                                  " Here are the details:\n \nName: $name \n ".
                                  "Email: $email_address\n Message \n $message";    

$mail_options = [
	"sender" => $email_address,
	"to" => $to,
	"subject" => $email_subject,
	"textBody" => $email_body
	
$mail_options = [
    "sender" => "admin@example.com",
    "to" => "me@ulysseszheng0.appspotmail.com",
    "subject" => "Your example.com account has been activated.",
    "textBody" => "test 1 2 3"
];

try {
	$message = new Message($mail_options);
	$message->send();
	return true;                        
} catch (InvalidArgumentException $e) {
	return false;
}


?>

File: contact_me.py
from google.appengine.ext import webapp
from google.appengine.api import mail
from google.appengine.api import users

from google.appengine.ext import webapp
from google.appengine.ext.webapp.util import run_wsgi_app
import logging
import wsgiref.handlers

class InviteFriendHandler(webapp.RequestHandler):
	def post(self):

		from_addr = self.request.get("email")

		message = mail.EmailMessage()
		message.sender = "me@ulysseszheng0.appspotmail.com"
		message.to = "ulysses147@gmail.com"
		message.body = "Sent from: %s.\nBody:\n%s" % (from_addr, self.request.get("message"))
		message.subject = "Message from: %s" % from_addr
		logging.info("sending to %s" % message.sender)
		message.send()
	def get(self):
		logging.info("sdfasfdasfas Hello get")
def main():
	application = webapp.WSGIApplication([('/.*', InviteFriendHandler),
										('/contact_me.py', InviteFriendHandler)], debug=True)
	run_wsgi_app(application)


if __name__ == '__main__':
	main()